lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -56,6 +56,12 @@
 #define TEE_ATTR_PBKDF2_ITERATION_COUNT     0xF00003C2
 #define TEE_ATTR_PBKDF2_DKM_LENGTH          0xF00004C2
 
+/*
+ * PKCS#1 v1.5 RSASSA pre-hashed sign/verify
+ */
+
+#define TEE_ALG_RSASSA_PKCS1_V1_5	0x70007830
```

Comment:
This defines a value in a range that as far as I can tell is reserved for future specifications.

Reply:
@jenswi-linaro indeed this is a problem. The future Internal Core API specification v1.2 (currently numbered v1.1.2.51) gets rid of the structure of algorithm identifiers (table 6-12) and defines a whole range for implementation-defined identifiers (0xF0000000-0xF0FFFFFF). A value in that range would be an option I guess, but that certainly means more invasive changes to the OP-TEE code which currently assumes the structure defined in the current spec.

Reply:
@jenswi-linaro @jforissier then would `0xF0000830` be all right for this define?

Reply:
@EDVTAZ I think it should be OK.

Reply:
@EDVTAZ ...but `0xF0000001` would be even better.

Reply:
I've tried to make this work, but a lot of macros that are used all over the place rely on the identifier's value having the above mentioned structure. Most of the time the macros are called in switch statements, so I can't insert cases for an identifier that doesn't follow the structure. I've inserted if-else statements before switch statements to get around this, but it seems I'm missing something somewhere, because it fails to work. The resulting code is also ugly and hard to read.

Could we maybe find a value that works with the current macros, or just keep the current value until the changes @jforissier mentioned get implemented?

Reply:
@jforissier @EDVTAZ  , just a gentle reminder about this, since it seems like the discussion suddenly stopped (a since I also think it is a useful addition).